packages/interfaces/src/kobo/PatchDroughtFormDto.ts

```diff
@@ -0,0 +1,94 @@
+import { IsDateString, IsIn, IsNumberString, IsOptional, IsString, Length } from 'class-validator';
+
+export class PatchDroughtFormDto {
+  @IsOptional() @IsString() readonly 'group_ve4vz14/q_Enum'!: string;
+  @IsOptional() @IsDateString() readonly 'group_ve4vz14/Date_report'!: string;
+  @IsOptional() @IsString() readonly 'group_ve4vz14/q_Funtion'!: string;
+  @IsOptional() @IsString() readonly 'group_ve4vz14/q_Phone'!: string;
+  @IsOptional() @IsString() @Length(2) readonly 'group_yu9nq00/Province'!: string;
+  @IsOptional() @IsString() @Length(4) readonly 'group_yu9nq00/District'!: string;
+  @IsOptional() @IsString() @Length(6) readonly 'group_yu9nq00/Commune'!: string;
+  @IsOptional() @IsDateString() readonly 'group_yu9nq00/Date_Dis': string;
+  @IsOptional() @IsNumberString() readonly 'group_yu9nq00/DisTyp'!: string;
```

Comment:
Do we know if the group ids will change? This feels like it will break easily. Do we have a more future-proof way to do this?